docs/src/main/asciidoc/grpc-getting-started.adoc

```diff
@@ -34,6 +34,17 @@ Edit the `pom.xml` file to add the Quarkus gRPC extension dependency (just under
 </dependency>
 ----
 
+Reactive REST endpoints in your application that return Uni or Multi need `Mutiny support for RESTEasy` extension (`io.quarkus:quarkus-resteasy-mutiny`) to work properly,
+add this to the dependencies section of your `pom.xml` file:
+
+[source,bash]
+----
+<dependency>
+    <groupId>io.quarkus</groupId>
+    <artifactId>quarkus-resteasy-mutiny</artifactId>
+</dependency>
```

Comment:
@michalszynkiewicz is this guide using RESTEasy Classic or Reactive? If it's not the latter, then I think it should be changed

Reply:
So the guide starts with:
```
Let's imagine you have a regular Quarkus project, generated from the https://code.quarkus.io
```

By default we do classic RESTEasy. I think we should change the guide. Either to encourage users to add RESTEasy Reactive to their project at first or to generate the project with both, `resteasy-reactive` and `grpc` from the start.
Which do you think would be preferable? I see both ways in the guides we have.

Reply:
I don't really mind to be honest

Reply:
Changed for `resteasy-reactive`. Generating project with grpc right away will apply codestart for grpc, which will require changing the bigger part of the guide.